packit/api.py

```diff
@@ -63,6 +63,7 @@ def __init__(
 
         self._up = None
         self._dg = None
+        self._create_pr = None
```

Comment:
this is overkill, we don't really need to store the value in a new variable; why not just access self.package_config.create_pr?

Reply:
Reason was to have `self.create_pr` as a property. Nothing else.

Reply:
but why?

Reply:
Because we don't instantiate any class like this (https://github.com/packit-service/packit/blob/master/packit/api.py#L70) then it does not make sense. Will change it.